src/main/scala/edu/twitter/model/impl/gradientboosting/GradientBoostingModel.scala

```diff
@@ -13,10 +14,9 @@ class GradientBoostingModel(model: GradientBoostedTreesModel) extends GenericMod
 
   val name: String = GradientBoostingModel.name
   private val hashingTF = new HashingTF(2000)
-  private val invalidTokens = Set("http", "@", "rt", "#", "RT")
 
   override def getLabel(tweetText: String): Double = {
-    val tokens = tweetText.split(" ").filter(!invalidTokens(_))
+    val tokens = TweetTextFilter.filterTweet(tweetText).split(" ")
```

Comment:
Could you please do the same for the neural network model?
Also, Don't you think we need to filter both the training and the testing data before training the model in the same manner?

Reply:
Sure will do it. I think this could improve the accuracy because URLs and mentions are not relevant keywords and we use can't them in the classification.

Reply:
Yeah, but we still need to work on that more, for example, you might be interested in knowing that the tweet contains a URL and in this case, it would be better to replace the URL with a simple keyword such as "@URL". Dr.Marwan mentioned that there might be some libraries that could do the work for us. Ramadan & I tried to look for one but we didn't find anything till now, maybe you could give it a try later.

Reply:
Sorry, I'm not following you. Whey you would replace it with `@url` I guess you need to remove it from the tweet text